cranelift/fuzzgen/src/config.rs

```diff
@@ -86,6 +88,7 @@ impl Default for Config {
             switch_max_range_size: 2..=32,
             static_stack_slots_per_function: 0..=8,
             static_stack_slot_size: 0..=128,
+            stack_slot_alignment_log2: 0..=10,
```

Comment:
Currently the max working alignment is 2^4 (16 bytes): https://github.com/bytecodealliance/wasmtime/pull/8635#discussion_r1603954465

Reply:
👍 I've been following that thread. We currently don't depend on any alignment larger than 8 bytes. And at most we'll need 16bytes (for sinking SIMD loads).

But I think it's still a good idea to test with larger values, even if we end up not having any guarantee past 16 bytes.

Reply:
I missed that you are not actually checking that the stack slot is correctly aligned at runtime.